aiida/cmdline/commands/cmd_code.py

```diff
@@ -172,25 +172,32 @@ def show(code, verbose):
 
 @verdi_code.command()
 @arguments.CODES()
+@options.FORCE(help='Force deletion of code and all calculations using it.')
```

Comment:
I could add a `NON_INTERACTIVE`  option as well.

One small issue is that for `verdi node delete`, the option to suppress prompts is called `--force`.
I think this should actually be called `--non-interactive` as well.

Reply:
Another option is: I don't modify `verdi code delete` at all, except for adding the info message:
`echo.echo_info("Use 'verdi node delete <pk>' to delete the code and all of its calculations")`.